src/types/hermezjs.d.ts

```diff
@@ -624,10 +618,17 @@ declare module "@hermeznetwork/hermezjs/src/api" {
   // function getCreateAccountAuthorization();
   // function getConfig();
 
-  function getPoolTransactions(
-    accountIndex: string,
-    publicKeyCompressedHex: string
-  ): Promise<HistoryTransaction[]>;
+  // function getPoolTransactions(
```

Comment:
Why do we have two `getPoolTransactions` declarations?

Reply:
Because the library has two implementations, one in the [module `api`](https://github.com/hermeznetwork/hermezjs/blob/main/src/api.js#L152) and another in the [module `tx-pool`](https://github.com/hermeznetwork/hermezjs/blob/main/src/tx-pool.js#L35).

Reply:
Oh right! I forgot about the old implementation in the `tx-pool` module hehe